app/src/main/java/gov/hhs/cdc/trustedintermediary/organizations/Organization.java

```diff
@@ -0,0 +1,14 @@
+package gov.hhs.cdc.trustedintermediary.organizations;
+
+public class Organization {
```

Comment:
Also, please add a unit test for this.  It's super simple.  You can copy the concept from `app/src/test/groovy/gov/hhs/cdc/trustedintermediary/domainconnector/DomainRequestTest.groovy`.